Export API to v1.0 (#21700)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/security-ediscoveryexportoperation.md

---
title: "ediscoveryExportOperation resource type"
description: "Represents the process of an eDiscovery export."
author: "vbhatt0211"
ms.localizationpriority: medium
ms.prod: "ediscovery"
doc_type: resourcePageType
---

# ediscoveryExportOperation resource type

Namespace: microsoft.graph.security

Represents the process of an eDiscovery export.

Inherits from [caseOperation](../resources/security-caseoperation.md).

## Methods
|Method|Return type|Description|
|:---|:---|:---|

> |[Get caseOperation](../api/security-caseoperation-get.md)|Resource | Please use the new **exportFileMetadata** property returned by the method to obtain downloadUrl,fileName and size of exported content |

Reviewer: ```suggestion
|[Get caseOperation](../api/security-caseoperation-get.md)|Resource | Please use the **exportFileMetadata** property returned by the method to obtain downloadUrl, fileName, and size of the exported content.|
```

> ## Properties

Reviewer: The properties in the CSDL, the api.md, and this file don't match. Please clarify. Thanks.

---
title: "ediscoveryExportOperation resource type"
description: "Represents the process of an eDiscovery export."
author: "vbhatt0211"
ms.localizationpriority: medium
ms.prod: "ediscovery"
doc_type: resourcePageType
---

# ediscoveryExportOperation resource type

Namespace: microsoft.graph.security

Represents the process of an eDiscovery export.

Inherits from [caseOperation](../resources/security-caseoperation.md).

## Methods

|Method|Return type|Description|
|:---|:---|:---|

> |[Get caseOperation](../api/security-caseoperation-get.md)|Resource | Read the properties and relationships of a [caseOperation](../resources/security-caseoperation.md) object. Please use the new **exportFileMetadata** property returned by the method to obtain downloadUrl,fileName and size of exported content. |

Reviewer 2: @vbhatt0211, is the following sentence required for this v1.0 version. I think that for the beta scenario it was relevant but there hasn't been any other option in v1.0 to obtain the following properties: downloadUrl, fileName and size. If this sentence is not needed, please remove it.

- Please use the new **exportFileMetadata** property returned by the method to obtain downloadUrl,fileName and size of exported content.

Author: Updated the statement as  - "The **exportFileMetadata** property returned by the method will provide downloadUrl,fileName and size of exported content " 
This is new property json which will provide these details. Hence adding in description

---
title: "ediscoveryExportOperation resource type"

> description: "Represents the process of an eDiscovery export."

Reviewer: ```suggestion
description: "Represents the process of a Microsoft Purview eDiscovery export."
```

author: "vbhatt0211"
ms.localizationpriority: medium
ms.prod: "ediscovery"
doc_type: resourcePageType
---

> # ediscoveryExportOperation resource type

Reviewer: Please add this resource to the TOC.

Reviewer: @Lauragra, the author mentioned: I have not added this as the export is still not an independent resource; export is being used through review set and review set query. Advise me on what should be the ideal scenario.

Namespace: microsoft.graph.security

> Represents the process of an eDiscovery export.

Reviewer: ```suggestion
Represents the process of a Microsoft Purview eDiscovery export.
```

Inherits from [caseOperation](../resources/security-caseoperation.md).

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[Get caseOperation](../api/security-caseoperation-get.md)|Resource | The **exportFileMetadata** property returned by the method will provide downloadUrl, fileName and size of exported content |

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |action|[microsoft.graph.security.caseAction](../resources/security-caseoperation.md#caseaction-values)| The type of action the operation represents. Possible values are: `addToReviewSet`,`applyTags`,`contentExport`,`convertToPdf`,`estimateStatistics`, `purgeData`|

Reviewer: I reviewed the operations and the api.md file indicates that the following properties (`outputName`, `description`, `outputFolderId`, `azureBlobContainer`, `azureBlobToken`, `exportOptions`, and `exportStructure`) are inherited from another resource, please add the sentence "Inherited from (resource name)" next to each property description.

|id|String| The ID for the operation. Read-only. |
|outputName|String| The name provided for the export.|
|percentProgress|Int32| The progress of the operation.|
|resultInfo|[resultInfo](../resources/resultinfo.md)|Contains success and failure-specific result information. Inherited from [caseOperation](../resources/security-caseoperation.md).|
|status|[microsoft.graph.security.caseOperationStatus](../resources/security-caseoperation.md#caseoperationstatus-values)| The status of the case operation. Possible values are: `notStarted`, `submissionFailed`, `running`, `succeeded`, `partiallySucceeded`, `failed`.|

### exportOptions values

|Member|Description|
|:----|-----------|
|originalFiles|Include original files in native format. For example: docx, xlsx, pptx, doc, xlst, pptm, etc.|
|text|Include extracted text from the original files.|
|pdfReplacement|Replace original file with PDF version when available.|
|tags|Include tag information.|

### exportFileStructure values

|Member|Description|
|:----|-----------|
|None|Default file structure.|

> |directory|All files in a singe folder called NativeFiles.|

Reviewer: ```suggestion
|directory|All files in a single folder called NativeFiles.|
```
